content/app/design/guidelines/components/select/_index.en.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Nedtrekksliste
-description: Nedtrekksliste brukes i tilfeller der brukeren skal velge et alternativ blant flere. Det samme gjelder radioknapper, så vær bevisst på når du bruker hvilken.  
+title: Pull-down list
```

Comment:
This component is named as `Dropdown` in code, so we should be consistent with that in documentation. There are more references throughout this document.

```suggestion
title: Dropdown
```